features/step_definitions/authentication_steps.rb

```diff
@@ -0,0 +1,27 @@
+Given(/^I have an account on the site$/) do
+  FactoryGirl.create(:user, email: "user@example.com", password: "password")
+end
+
+Given(/^My account is confirmed$/) do
+  User.last.confirm
```

Comment:
You might want to look into using instance variables for this. Take a look at the style guide we have up (let me know if you can't find it). You would use that "I have an account" step to set an @user instance variable, and then ahve acess to it in the subsequent steps in that scenario.